content/en/docs/tasks/security/dns-cert/index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Istio DNS Certificate Management
+description: Shows how to provision and manage DNS certificates in Istio.
+weight: 10
+keywords: [security,certificate]
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+By default, the DNS certificates of Galley and Sidecar
+Injector are provisioned and managed by Citadel, which is a large component
+that maintains its own signing key and also acts as a CA for Istio.
+This task shows how to provision and manage DNS certificates in Istio through
+a lightweight component (called Chiron), which signs certificates
+using Kubernetes CA APIs without maintaining its own private key.
+
+## Before you begin
+
+* Create a Kubernetes cluster with Istio installed and the DNS certificate configuration
+in [`values-istio-dns-cert.yaml`]({{< github_file >}}/install/kubernetes/helm/istio/example-values/values-istio-dns-cert.yaml).
+Istio installation guides can be found [here](/docs/setup/install).
+
+* Install [`jq`](https://stedolan.github.io/jq/) for JSON parsing.
+
+## DNS certificate provisioning and management
+
+You can configure the DNS names and secret names for the DNS certificates
```

Comment:
In what cases does the user configure the DNS names and secret names? If it's not needed in the default use cases, maybe we should put this section and the "Configure DNS certificates" section to the end?

Reply:
This is explained in the beginning of the guide as follows:
By default, the DNS certificates of Galley and Sidecar
Injector are provisioned and managed by Citadel, which is a large component
that maintains its own signing key and also acts as a CA for Istio.
This task shows how to provision and manage DNS certificates in Istio through
a lightweight component (called Chiron), which signs certificates
using Kubernetes CA APIs without maintaining its own private key.